data/json/itemgroups/SUS/domestic.json

```
@@ -191,5 +191,27 @@
       { "item": "bleach", "prob": 75, "charges-min": 1 },
       { "item": "ammonia", "prob": 65, "charges-min": 1 }
     ]
+  },
+  {
+    "id": "SUS_pantry",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for a regular stocked kitchen pantry.",
+	"//3": "Frequency of pet food appearing based on percentage of Massachussets homes with a pet.",
```

```suggestion
	"//3": "Frequency of pet food appearing based on percentage of Massachusetts homes with a pet.",
```
unplayable without this change